pylint/checkers/typecheck.py

```diff
@@ -916,13 +916,6 @@ def visit_attribute(self, node):
 
         function/method, super call and metaclasses are ignored
         """
-        for pattern in self.config.generated_members:
```

Comment:
Can you keep these lines here? Checking these patterns are faster than doing the inference at line 920, so if there's a match, we won't have to trigger the inference in the first place.

Also it would be great if you can compile these patterns in `open()`, and use the pre-compiled patterns instead of calling `re.match`. I know it was like this before, but it would be a nice improvement!

Reply:
This comment was taken into account in #4092